packages/rpc-graphql/src/schema/instruction.ts

```diff
@@ -882,6 +882,22 @@ export const instructionTypeDefs = /* GraphQL */ `
         source: Account
     }
 
+    """
+    SplToken-2022: WithdrawConfidentialTransfer instruction
+    """
+    type SplTokenWithdrawConfidentialTransfer implements TransactionInstruction {
+        programId: Address
+        amount: BigInt
+        decimals: BigInt # FIXME:*
+        destination: Account
+        instructionsSysvar: Account
+        mint: Account
+        multisigOwner: Account
+        owner: Account
+        signers: [Address]
+        source: Account
+    }
```

Comment:
Missing two fields here.
https://github.com/anza-xyz/agave/blob/06ef0fb5bc2ae9ce4044ff90dbc70000c055f03c/transaction-status/src/parse_token/extension/confidential_transfer.rs#L168-L169

Reply:
Oops forgot to commit